doc/whats_new.rst

```diff
@@ -100,6 +100,10 @@ Enhancements
     - Extensive plotting example shows how to use contours and filled contours
       on glass brain.
 
+    - Function :func:`nilearn._utils.ndimage.largest_connected_component_img`
+      so that niftis can now be passed to 
+      :func:`nilearn._utils.ndimage.largest_connected_component`.
```

Comment:
Also, I could not understand why largest_connected_component is referred here. May be you could write "so that Nifti images can now be passed on directly to extract largest connected components in the images."